pkg/systemtemplate/template.go

```diff
@@ -218,6 +218,11 @@ spec:
         secret:
           secretName: cattle-credentials-{{.TokenKey}}
           defaultMode: 320
+  strategy:
+    type: RollingUpdate
+    rollingUpdate:
+      maxUnavailable: 0
+      maxSurge: 1
```

Comment:
This is `25%` by default and rounding down, so effectively we are allowing one additional pod here with this change, right?